pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -256,7 +256,7 @@ func (cm *containerManagerImpl) validateNodeAllocatable() error {
 		value.Sub(v)
 
 		if value.Sign() < 0 {
-			errors = append(errors, fmt.Sprintf("Resource %q has an allocatable of %v, capacity of %v", k, v, value))
+			errors = append(errors, fmt.Sprintf("Resource %q has an reservation of %v, capacity of %v", k, v, cm.capacity[k]))
```

Comment:
let's be even more explicit maybe? "Resource %q has a reservation of %v but capacity of %v. Expected capacity > reservation"

(also the net "an reservation" => "a reservation")